src/components/AppNavHeader.tsx

```diff
@@ -30,9 +30,10 @@ export function AppNavHeader({
     route,
     styles.titleText,
   );
+  const statusBarHeight = config.AppNavHeader?.statusBarHeight;
 
   return (
-    <Appbar.Header statusBarHeight={0} style={headerStyles}>
+    <Appbar.Header statusBarHeight={statusBarHeight} style={headerStyles}>
```

Comment:
The default header already includes safe-area-view.  That needs to be turned off dynamically when there is a logo header above it.

| Before | After |
|-|-|
| ![Screenshot 2023-05-31 at 9 55 12 AM](https://github.com/lifeomic/react-native-sdk/assets/220893/b9274207-716b-41cb-babe-4b3c9a5dfc28) | ![Screenshot 2023-05-31 at 9 55 38 AM](https://github.com/lifeomic/react-native-sdk/assets/220893/f4af68f7-6fdf-4cda-be15-c1a9cd2db7c1) |